addons/account_audit_trail/models/mail_message.py

```diff
@@ -3,18 +3,19 @@
 
 from markupsafe import Markup
 
-from odoo import fields, api, models, _
+from odoo import api, fields, models, _
 from odoo.exceptions import UserError
 
 
 class Message(models.Model):
     _inherit = 'mail.message'
 
-    l10n_in_audit_log_preview = fields.Html(string="Description", compute="_compute_l10n_in_audit_log_preview")
-    l10n_in_audit_log_account_move_id = fields.Many2one('account.move', string="Journal Entry", compute="_compute_l10n_in_audit_log_document_name", search="_search_l10n_in_audit_log_document_name")
+    account_audit_log_preview = fields.Html(string="Description", compute="_compute_account_audit_log_preview")
+    account_audit_log_account_move_id = fields.Many2one('account.move', string="Journal Entry", compute="_compute_account_audit_log_document", search="_search_account_audit_log_document_name")
+    show_audit_log = fields.Boolean(compute="_compute_account_audit_log_document", search="_search_show_audit_log")
```

Comment:
I don't understand why you need this field.
Can't you simply search for `('account_audit_log_account_move_id', '!=', False)`?

Reply:
@william-andre The ```show_audit_log``` field is necessary because it is used to check whether the ```Audit Trail``` option in settings is enabled or not.

Reply:
Why can't you include that in `account_audit_log_account_move_id`?